proxy.go

```diff
@@ -30,6 +31,10 @@ import (
 const (
 	proxyName  = "kata-proxy"
 	termSignal = syscall.SIGTERM
+	// Max number of threads the proxy should consume.
+	// We choose 6 as we want a couple of threads for the runtime (gc etc.)
+	// and couple of threads for our parallel user code copy routines.
+	maxThreads = 6
```

Comment:
uhmmm 6 threads for a POD with N containers (shims)...

Reply:
:-) I was hoping somebody might debate the '6'.
(you no doubt know this, but...) So, with 6 threads you can run 6 bits of code in parallel - like, really in parallel. You can have as many goroutines pending or sleeping and scheduled to execute. How many goroutines do we think we actually execute really in parallel, that are not blocked on IO or are that time critical? We are not afaik compute bound....
I'm more than happy to bump the number, but I'd just like to be sure we will actually use those parallel threads in parallel, and not just consume host PIDs for no real reason.